modules/tinymce/src/plugins/table/main/ts/actions/ResizeHandler.ts

```diff
@@ -118,8 +114,8 @@ export const getResizeHandler = function (editor: Editor): ResizeHandler {
     if (isTable(targetElm)) {
       const table = targetElm;
 
-      if (percentageBasedSizeRegex.test(startRawW)) {
-        const percentW = parseFloat(percentageBasedSizeRegex.exec(startRawW)[1]);
+      if (Util.isPercentage(startRawW)) {
+        const percentW = parseFloat(startRawW.replace('%', ''));
```

Comment:
These two lines could be broken out and unit tested.

Reply:
Out of scope for this JIRA, as it's not something I've added/changed.